python/emitter.js

```diff
@@ -398,10 +398,10 @@
         },
         close(throwEx = true) {
             let isThrow = throwEx.toString()
-            return addCommand(python`eyes.close(raise_ex=` + isThrow[0].toUpperCase() + isThrow.slice(1) + `)`)
+            return addCommand(python`eyes.close(raise_ex=` + isThrow[0].toUpperCase() + isThrow.slice(1) + `)`).type('TestResults')
         },
         abort() {
-            return addCommand(python`eyes.abort()`)
+            return addCommand(python`eyes.abort()`).type('TestResults')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9142511)
```suggestion
            return addCommand(python`eyes.abort()`).type("TestResults")
```